### PR TITLE
adding the promotion categories to the subnav

### PR DESCRIPTION
**Pull request by mgharbik**

### CODE REVIEW

File: backend/spec/features/admin/homepage_spec.rb [hunk 1]
```diff
@@ -51,6 +55,20 @@
...
       end
     end
+
+    context "visiting the promotions tab" do
+      before(:each) do
+        visit spree.admin_promotions_path
+      end
+
+      it "should have a link to promotions" do
+        within('.selected .admin-subnav') { page.find_link("Promotions")['/admin/promotions'] }
```
Comment (jhawthorn) [inline backend/spec/features/admin/homepage_spec.rb]: I don't think this `[]` syntax works (I see it elsewhere in the file, I don't think that does anything either).

```
page.find_link("Promotions")['/admin/promotions'] #=> nil, no spec failure
page.find_link("Promotions")['foooooooooo'] #=> nil, no spec failure
```

Instead can we make this

```
expect(page).to have_link("Promotions")
```

File: backend/spec/features/admin/homepage_spec.rb [hunk 2]
```diff
@@ -51,6 +55,20 @@
...
+      end
+
+      it "should have a link to promotion categories" do
+        within('.selected .admin-subnav') { page.find_link("Promotion Categories")['/admin/promotion_categories'] }
```
Comment (jhawthorn) [inline backend/spec/features/admin/homepage_spec.rb]: Same as above

**Comment by jhawthorn** [comment]
Other than the two spec changes this looks great. Thanks very much.

**Comment by simo163** [comment]
you're welcome! I pushed a new commit changing all the specs where the `[]` syntax appears. Sorry for these detected tabs, I modified the file in a console text editor and some lines were not aligned properly.

**Comment by mamhoff** [comment]
The promotion categories sub navigation has been added to the menu already. I've opened another PR with your test suite improvements.

**Pull request closed without merging**